tests/baseclass.py

```diff
@@ -46,6 +47,25 @@ def setUp(self):
         self._device = _ped.device_get(self.path)
         self.device = parted.getDevice(self.path)
 
+# Base class for any test case, which requires loop block device - need run
+# as root
+class RequiresLoopDisk(unittest.TestCase):
```

Comment:
I'm not a fan of introducing yet another base class unless absolutely necessary. At first look I think some of the others could be used instead of a new one. I will have to give it a try first though.